utils/tests/test_fields.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+""" Tests utils/parsers.py """
+import re
+
+from utils.fields import BaseAPIField, ArrayField, EnumField
+from utils.constants import DataType
+from utils.fake_data import MilMoveProvider, MilMoveData
+from faker.generator import Generator
+
+
+class TestBaseAPIField:
+    """ Tests the BaseAPIField class and its methods """
+
+    @classmethod
+    def setup_class(cls):
+        """ Initialize the BaseAPIField that will be tested. """
+        cls.field1 = BaseAPIField(DataType.STREET_ADDRESS, name="streetAddress")
+        cls.faker = MilMoveData()
+
+    def test_add_discriminator_value(self):
+        """
+        Tests a value is appended to the list of discriminator values.
+        """
+        testString = "test value"
```

Comment:
this is an annoying comment, but technically this should be snake case: `test_string`